src/lammpsio/snapshot.py

```diff
@@ -35,19 +36,25 @@ class Snapshot:
 
     """
 
-    def __init__(self, N, box, step=None):
+    def __init__(self, N, box, num_types, step=None):
```

Comment:
For backwards compatibility, we need to add this at the end of this constructor:
```suggestion
    def __init__(self, N, box, step=None, num_types=None):
```
We can move it in a future major version release if we think it's important to reorder.